libraries/AP_HAL_Linux/Util_RPI.cpp

```diff
@@ -27,33 +28,53 @@ extern const AP_HAL::HAL &hal;
 
 using namespace Linux;
 
-UtilRPI::UtilRPI()
-{
-    _get_board_type_using_peripheral_base ();
-}
-
+UtilRPI::UtilRPI() { _get_board_type_using_peripheral_base(); }
```

Comment:
Whitespace change which changes the style o be contrary to our project standards.

Not a good idea to make this sort of change unrelated to what you're trying to accomplish with a PR - sometimes you hit rules like this.